recipes/django-redis/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "django-redis" %}
+{% set version = "4.8.0" %}
+{% set sha256 = "5229da5b07ccb8d3e3e9ee098c0b7c03e20eba48634bc456697dd73d62c68b19" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - redis >=2.10.0
+  run:
+    - python
+    - django >=1.8
+    - redis >=2.10.0
```

Comment:
I think the issue you're having is that `redis` here refers to the actual `redis` package and not the python package that wraps it. I think you want to change `redis` here to `redis-py`, which will grab:

https://github.com/conda-forge/redis-py-feedstock

instead of:

https://github.com/ContinuumIO/anaconda-recipes/blob/master/redis/meta.yaml